torch-neuronx/training/fsdp_gpt2/setup_gpt2_model_dataset.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This file is actually for our pipeline setup (to grab the cache from s3), may not need it for external customer